src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,52 @@
+class Calculator {
+    fun main(string: String?): Double {
```

Comment:
좋은 코드란 무엇일까요?

저는 개인적으로 좋은 코드의 원칙중 하나인 "가독성이 좋아 이해하기 쉬운 코드"가 중요하다고 생각합니다! 
개발을 하면서, 코드를 짜는 시간보다, 읽는 시간이 많기때문에 가독성은 매우 중요하다고 생각이 들어요! 
```suggestion
    fun calculate(inputExpression: String?): Double {
```
변수나 함수에 조금더 의미있거 구체적인 이름들을 붙여주면 어떨까요?